pyutils/perftest/references/daint_nvcc_cray/128.json

```diff
@@ -5,8130 +5,9842 @@
-        0.00082642,
-        0.000844941,
-        0.000828752,
-        0.000841651,
-        0.000829455,
-        0.000840304,
-        0.000828739,
-        0.000830059,
-        0.000842276,
-        0.000830109,
-        0.000823678,
-        0.000827508,
-        0.000843916,
-        0.000832714,
-        0.000823318,
-        0.00083225,
-        0.000831141,
-        0.000828512,
-        0.000826929,
-        0.000845567,
-        0.000834675,
-        0.000830615,
-        0.000833453,
-        0.000848063,
-        0.000828207,
-        0.000830921,
-        0.000826566,
-        0.000833775,
-        0.000827265
+        0.000858845,
+        0.000862925,
+        0.000871501,
+        0.000868335,
+        0.000864975,
+        0.000858762,
+        0.000861667,
+        0.000884933,
+        0.00085625,
+        0.000870443,
+        0.000861322,
+        0.000877239,
+        0.000860292,
+        0.000866631,
+        0.000860722,
+        0.000863177,
+        0.00088329,
+        0.000866265,
+        0.00086368,
+        0.00086284,
+        0.000874329,
+        0.000861102,
+        0.000865095,
+        0.000864855,
+        0.000858883,
+        0.000878974,
+        0.000858929,
+        0.000867083,
+        0.000859882,
+        0.000874963,
+        0.000863573,
+        0.000862421,
+        0.000859514,
+        0.000872023,
+        0.000858909,
+        0.000881163,
+        0.000860451,
+        0.000862929,
+        0.000864514,
+        0.000863942,
+        0.000879122,
+        0.000895302,
+        0.000859682,
+        0.000857149,
+        0.000871428,
+        0.000867472,
+        0.000862048,
+        0.000862059,
+        0.000864928,
+        0.000881445,
+        0.000862031,
+        0.00086448,
+        0.00085782,
+        0.000884294,
+        0.000865804,
+        0.000860996,
+        0.000871872,
+        0.000868603,
+        0.000877801,
+        0.000866842,
+        0.000862768,
+        0.00085871,
+        0.000882015,
+        0.000868617,
+        0.000863545,
+        0.000857278,
+        0.000860595,
+        0.000880298,
+        0.000865906,
+        0.000861599,
+        0.000862412,
+        0.000882557,
+        0.000854254,
+        0.000869561,
+        0.000863029,
+        0.000869455,
+        0.000881168,
+        0.000863506,
+        0.000856402,
+        0.000862166,
+        0.000878349,
+        0.000864765,
+        0.000863763,
+        0.00086375,
+        0.00087382,
+        0.000864663,
+        0.000861365,
+        0.000861392,
+        0.000864368,
+        0.000882963,
+        0.000863437,
+        0.00086258,
+        0.000856213,
+        0.000882228,
+        0.000861704,
+        0.000867676,
+        0.000857861,
+        0.000868935,
+        0.000881427,
+        0.000863241
       ]
     },
     {
       "name": "horizontal_diffusion",
       "backend": "cpu_ifirst",
       "float_type": "float",
       "series": [
-        0.00048747,
```

Comment:
I think we should update the reference for this test again. In the perftest run that you did in the PR, we are back to the original (better numbers).

Reply:
Better now or not?